nav2_regulated_pure_pursuit_controller/README.md

```diff
@@ -63,6 +63,9 @@ Mixing the proximity and curvature regulated linear velocities with the time-sca
 | `max_allowed_time_to_collision` | The time to project a velocity command to check for collisions | 
 | `use_regulated_linear_velocity_scaling` | Whether to use the regulated features for curvature | 
 | `use_cost_regulated_linear_velocity_scaling` | Whether to use the regulated features for proximity to obstacles | 
+| `cost_scaling_dist` | The minimum distance from an obstacle to trigger the scaling of linear velocity, if `use_cost_regulated_linear_velocity_scaling` is enabled. The value set should be smaller than the `inflation_radius` set in the inflation layer of costmap, since inflation is used to compute the distance from obstacles | 
```

Comment:
```suggestion
| `cost_scaling_dist` | The distance from an obstacle to trigger the scaling of linear velocity, if `use_cost_regulated_linear_velocity_scaling` is enabled. The value set should be smaller or equal to the `inflation_radius` set in the inflation layer of costmap, since inflation is used to compute the distance from obstacles | 
```